ReactPeerProgram/hpcard/src/components/atoms/Icon/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+interface IconProps {
+  src: string;
+  alt: string;
+  height?: number | string;
+  width?: number | string;
+}
+
+const Icon: React.FC<IconProps> = ({ src, alt, height, width }) => {
+  return <img src={src} alt={alt} height={height} width={width} />;
```

Comment:
use id and class name and write the styles in css file or use styled component instead of height and width